### DISCUSSION TITLE
파이어폭스 유블럭 오리진에서 필터추가 방법이 뭔가요?

### DISCUSSION THREAD

**Discussion by IIIllIIIlll** [List-KR/community#68] [General]
<a href="https://ibb.co/6nsss4j"><img src="https://i.ibb.co/C1000nR/123.png" alt="123" border="0"></a>

파이어폭스용 유블럭 오리진 사용중입니다.
필터목록에서 불러오기 창을 열어 필터 주소를 입력하고 변경사항 적용까지 눌렀는데도 0개 중 0개 사용으로 나옵니다.
내용 보기를 눌렀을때 404: not found 라고 입력돼있습니다.
제가 뭘 잘못 사용하고 있는지 알 수 있을까요?

**Comment by piquark6046** [comment]
Firefox에서 <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>Delete</kbd> 를 눌러서 캐시 삭제를 진행해보세요.

**Reply by IIIllIIIlll** [reply]
캐시삭제 후에도 안돼서 재설치까지 진행하니까 정상적으로 나오네요. 감사합니다
